features/api.feature

```diff
@@ -72,14 +86,15 @@ Feature: DrupalContext
     And follow "First article with fields"
     Then I should see the text "PLACEHOLDER BODY"
 
+  @d7 @d8
   Scenario: Create and view a node with fields
-    Given I am viewing an "Article":
+    Given I am viewing an "article":
     | title | My article with fields! |
```

Comment:
It seems to have worked fine in D7. I'll fix it in this PR and add test coverage for both the machine names and the human readable names so this won't regress.